src/api/app/models/patchinfo.rb

```diff
@@ -199,14 +199,14 @@ def create_patchinfo(project, pkg_name, opts = {})
     { targetproject: @pkg.project.name, targetpackage: @pkg.name }
   end
 
-  def cmd_update_patchinfo(project, package)
+  def cmd_update_patchinfo(project, package, message = 'updated via updatepatchinfo call')
```

Comment:
In the message, `updatepatchinfo` should be `fetch_issues` since it was changed in the controller.

Reply:
`cmd_update_patchinfo` is called from both API and Webui controllers. In the API endpoint it's used like: `# POST /source/<project>/<package>?cmd=updatepatchinfo`and we don't want to change the name there. For this reason, we pass a different message using the name `fetch_issues`if it comes from the webui or `updatepatchinfo`if it comes from the API.